v2/emailpassword/common-customizations/sessions/session-verification-in-api/get-session.mdx

```diff
@@ -180,17 +180,19 @@ export default async function likeComment(req: SessionRequest, res: any) {
 ```tsx
 import { NextResponse, NextRequest } from "next/server";
 import SuperTokens from "supertokens-node";
-import { withSession } from "supertokens-node/nextjs";
+import { withPreParsedRequestResponse } from "supertokens-node/nextjs";
+import { CollectingResponse, PreParsedRequest } from "supertokens-node/framework/custom";
+import Session from "supertokens-node/recipe/session";
 // @ts-ignore
 import { backendConfig } from "@/app/config/backend";
 
 SuperTokens.init(backendConfig());
 
 export function POST(request: NextRequest) {
-    return withSession(request, async (session) => {
-        let userId = session!.getUserId();
-        //....
-        return NextResponse.json({})
+    return withPreParsedRequestResponse(request, async (baseRequest: PreParsedRequest, baseResponse: CollectingResponse) => {
+        const session = await Session.getSession(baseRequest, baseResponse);
```

Comment:
you need to catch errors from this and call the errorHandler exposed from our SDK. See what `superTokensNextWrapper` does for pages dir

Reply:
done